pages/common/aws-rds.md

```diff
@@ -0,0 +1,37 @@
+# aws rds
+
+> CLI for AWS RDS.
+> Create and manage relational databases.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/index.html>.
+
+- Show help for specific RDS subcommand:
+
+`aws rds {{subcommand}} help`
+
+- Stop instance:
+
+`aws rds stop-db-instance --db-instance-identifier {{instance_identifier}}`
+
+- Start instance:
+
+`aws rds start-db-instance --db-instance-identifier {{instance_identifier}}`
+
+- Modify an RDS instance:
+
+`aws rds modify-db-instance --db-instance-identifier {{instance_identifier}} {{parameters}} --apply-immediately`
```

Comment:
this `parameters` placeholder is too generic, maybe example about modify an instance characteristic is better.